### PR TITLE
Developer's Guide to Choosing the Right API for Global Payouts

### PR DESCRIPTION
**Pull request by ayan-joshi**
Added the required article
Closing issue #384 

Have a look @phyleria

### CODE REVIEW

**Comment by phyleria** [comment]
Ok ✅

File: submissions/Dev focused articles/GlobalPayoutGuide.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+- [Payout Bank API Documentation](https://chimoney.readme.io/reference/post_v0-2-payouts-bank-1)
+- [Supported Bank Countries](https://chimoney.readme.io/reference/get_v0-2-info-country-banks-1)
+
+### 5. **Gift Card Payouts**
+Chimoney offers **gift card payouts** in over 20 countries, with more than 200 gift card options. This is a great solution for businesses looking to reward users with popular gift card options in various regions.
+
+- [Payout Gift Card API Documentation](https://chimoney.readme.io/reference/post_v0-2-payouts-gift-card-1)
+
+### 6. **XRPL Payments**
```
Comment (phyleria) [inline submissions/Dev focused articles/GlobalPayoutGuide.md:84]: Interledger Payment Pointers for Web Monetization.

File: submissions/Dev focused articles/GlobalPayoutGuide.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Chimoney supports **XRPL (XRP Ledger) payments**, enabling businesses to leverage blockchain technology for fast and secure payouts. This feature provides an innovative way to handle digital asset transactions on a global scale.
+
+- [Payout XRPL API Documentation](https://chimoney.readme.io/reference/post_v0-2-payouts-initiate-chimoney-1)
+
+---
+
+## Getting Started with the Chimoney API
```
Comment (phyleria) [inline submissions/Dev focused articles/GlobalPayoutGuide.md:91]: Please reference from this article on `Getting Started with the Chimoney API` part - https://chimoney.io/blogs/the-ultimate-guide-to-simplifying-global-payouts-for-your-business-with-api-solutions/

**Comment by ayan-joshi** [comment]
@phyleria Made required changes have a look if approved , I'll go to the next one , Thanks !

**Comment by ayan-joshi** [comment]
> @phyleria Made required changes have a look if approved , I'll go to the next one , Thanks !

@phyleria have a look

**Comment by ayan-joshi** [comment]
> @phyleria Made required changes have a look if approved , I'll go to the next one , Thanks !

@phyleria have a look

File: submissions/Dev focused articles/GlobalPayoutGuide.md [hunk 3]
```diff
@@ -81,15 +81,17 @@ Chimoney offers **gift card payouts** in over 20 countries, with more than 200 g
...
-- [Payout XRPL API Documentation](https://chimoney.readme.io/reference/post_v0-2-payouts-initiate-chimoney-1)
+- [Payout API Documentation](https://chimoney.readme.io/reference/post_v0-2-payouts-initiate-chimoney-1)
 
 ---
 
 ## Getting Started with the Chimoney API
 
+Chimoney's Global Payouts API provides a seamless way for businesses to execute global payments effortlessly. This guide is tailored for developers, featuring step-by-step instructions on how to integrate Chimoney's API. Let's dive into the services you can harness the power of Chimoney's Global Payouts API to enhance your payment infrastructure. You will just need Chimoney account with API access , [Get Sandbox Access here](https://chimoney.readme.io/reference/sandbox-environment).
+
 ### Services Offered by Chimoney API:
```
Comment (phyleria) [inline submissions/Dev focused articles/GlobalPayoutGuide.md:95]: Kindly remove this part

**Review by phyleria — CHANGES_REQUESTED** [review]
Under other 'Global Payout solutions, kindly remove Stripe. Also please link Paypal, Wise and Payoneer to the comparison articles we have on the footer section at [chimoney.io](https://chimoney.io/) as attached 
image

**Review by phyleria — CHANGES_REQUESTED** [review]
Kindly replace this `For more information on how to get started with Chimoney, visit the [Chimoney API Documentation](https://chimoney.io). with this `Interested in learning more about how the Chimoney API can streamline your payouts? Book a [demo session](https://chimoney.io/?book_a_demo=1) with us today.`

File: submissions/Dev focused articles/GlobalPayoutGuide.md [hunk 4]
```diff
@@ -81,15 +81,17 @@ Chimoney offers **gift card payouts** in over 20 countries, with more than 200 g
...
+Chimoney supports **Interledger Payment Pointers for Web Monetization**, enabling businesses to leverage blockchain technology for fast and secure payouts. This feature provides an innovative way to handle digital asset transactions on a global scale.
 
-- [Payout XRPL API Documentation](https://chimoney.readme.io/reference/post_v0-2-payouts-initiate-chimoney-1)
+- [Payout API Documentation](https://chimoney.readme.io/reference/post_v0-2-payouts-initiate-chimoney-1)
 
 ---
 
 ## Getting Started with the Chimoney API
 
+Chimoney's Global Payouts API provides a seamless way for businesses to execute global payments effortlessly. This guide is tailored for developers, featuring step-by-step instructions on how to integrate Chimoney's API. Let's dive into the services you can harness the power of Chimoney's Global Payouts API to enhance your payment infrastructure. You will just need Chimoney account with API access , [Get Sandbox Access here](https://chimoney.readme.io/reference/sandbox-environment).
```
Comment (phyleria) [inline submissions/Dev focused articles/GlobalPayoutGuide.md:93]: Please update the following section to the following steps and their description:
- Get Sandbox Access
- Test Your Integration
- Go Live!

**Comment by ayan-joshi** [comment]
@phyleria made the changes , have a look !

File: submissions/Dev focused articles/GlobalPayoutGuide.md [hunk 5]
```diff
@@ -43,9 +43,13 @@ Chimoney provides a comprehensive global payout API that offers unmatched flexib
 While there are many payout APIs in the market, Chimoney's flexibility and global reach make it stand out. However, here’s how some other solutions compare:
 
 - **PayPal** offers global payouts, but primarily focuses on traditional payment methods like bank transfers and digital wallets, lacking support for mobile money and airtime.
-- **Stripe** is known for its payment processing capabilities, but it does not offer as many options for payout diversity, such as airtime or gift cards.
```
Comment (phyleria) [inline submissions/Dev focused articles/GlobalPayoutGuide.md]: Hi @adarsh-jha-dev 

Please add the comparison links in this format `**[Stripe]((https://chimoney.io/using/paypal/for-sending/mass-payments/vs-using/chimoney/)** is known for its payment processing capabilities, but it does not offer as many options for payout diversity, such as airtime or gift cards.`

File: submissions/Dev focused articles/GlobalPayoutGuide.md [hunk 6]
```diff
@@ -90,21 +94,21 @@ Chimoney supports **Interledger Payment Pointers for Web Monetization**, enablin
 
 ## Getting Started with the Chimoney API
 
-Chimoney's Global Payouts API provides a seamless way for businesses to execute global payments effortlessly. This guide is tailored for developers, featuring step-by-step instructions on how to integrate Chimoney's API. Let's dive into the services you can harness the power of Chimoney's Global Payouts API to enhance your payment infrastructure. You will just need Chimoney account with API access , [Get Sandbox Access here](https://chimoney.readme.io/reference/sandbox-environment).
+Chimoney's Global Payouts API provides a seamless way for businesses to execute global payments effortlessly. This guide is tailored for developers, featuring step-by-step instructions on how to integrate Chimoney's API. Let's dive into the services you can harness the power of Chimoney's Global Payouts API to enhance your payment infrastructure. You will just need Chimoney account with API access.
```
Comment (phyleria) [inline submissions/Dev focused articles/GlobalPayoutGuide.md:97]: For this part, please change it to `To get started with the Chimoney API, follow this process:`

**Comment by ayan-joshi** [comment]
@phyleria made changes !

File: submissions/Dev focused articles/GlobalPayoutGuide.md [hunk 7]
```diff
@@ -95,7 +92,7 @@ Chimoney supports **Interledger Payment Pointers for Web Monetization**, enablin
 ## Getting Started with the Chimoney API
```
[the remark this answers is not in the document]
Comment (ayan-joshi) [reply submissions/Dev focused articles/GlobalPayoutGuide.md:93]: okay

**Comment by ayan-joshi** [comment]
@phyleria its done

**Comment by CBID2** [comment]
Hey @ayan-joshi. When you link and issue(s) that correspond to your pull request(s), I highly recommend doing this 

```md
## Issue
Closes Issue number (e.g., #0)
```
This will help maintainers manage their issues as it automatically closes PRs.

**Comment by ayan-joshi** [comment]
> Hey @ayan-joshi. When you link and issue(s) that correspond to your pull request(s), I highly recommend doing this
> 
> ```
> ## Issue
> Closes Issue number (e.g., #0)
> ```
> 
> This will help maintainers manage their issues as it automatically closes PRs.

On the top , I have mentioned the issue it will close @CBID2

**Comment by CBID2** [comment]
> > Hey @ayan-joshi. When you link and issue(s) that correspond to your pull request(s), I highly recommend doing this
> 
> > 
> 
> > ```
> 
> > ## Issue
> 
> > Closes Issue number (e.g., #0)
> 
> > ```
> 
> > 
> 
> > This will help maintainers manage their issues as it automatically closes PRs.
> 
> 
> 
> On the top , I have mentioned the issue it will close @CBID2 

I know @ayan-joshi , but the way you currently posted your pull request's number won't appear in the Development section (see attached photo below👇)

image

Doing it in the way I mentioned helps with this. Does that clarify things a bit more now?

**Comment by ayan-joshi** [comment]
> > > Hey @ayan-joshi. When you link and issue(s) that correspond to your pull request(s), I highly recommend doing this
> > 
> > 
> > > 
> > 
> > 
> > > 
> > 
> > 
> > > ## Issue
> > 
> > 
> > > Closes Issue number (e.g., #0)
> > 
> > 
> > > 
> > 
> > 
> > > 
> > 
> > 
> > > This will help maintainers manage their issues as it automatically closes PRs.
> > 
> > 
> > On the top , I have mentioned the issue it will close @CBID2
> 
> I know @ayan-joshi , but the way you currently posted your pull request's number won't appear in the Development section (see attached photo below👇)
> 
> image
> 
> Doing it in the way I mentioned helps with this. Does that clarify things a bit more now?

Okay got it thanks !

**Review by phyleria — APPROVED** [review]

**Comment by phyleria** [comment]
Thank you so much @ayan-joshi for your work on this 🎉

**Comment by ayan-joshi** [comment]
> Thank you so much @ayan-joshi for your work on this 🎉

Thanks to you as well @phyleria

**Comment by ayan-joshi** [comment]
@phyleria By mistake I added images in here now removed successfully have a look please to approve and merge the pr

**Review by phyleria — APPROVED** [review]

**Pull request merged**